apps/election-manager/src/screens/TallyScreen.tsx

```diff
@@ -389,7 +357,9 @@ const TallyScreen: React.FC = () => {
           <React.Fragment>
             <h2>Export Options</h2>
             <p>
-              <Button onPress={exportResults}>Export Results File</Button>
+              <Button onPress={() => setIsExportResultsModalOpen(true)}>
+                Export Results File
```

Comment:
I'm thinking we should stop calling this "export" and start calling it "Save" results (in the user text, the React components can of course be called `export`)